test/model/src/driver/debug_log_parser.cpp

```diff
@@ -536,10 +582,39 @@ from_debug_log_helper(WT_SESSION_IMPL *session, WT_ITEM *rawrec, WT_LSN *lsnp, W
         break;
     }
 
+    case WT_LOGREC_SYSTEM: {
```

Comment:
New line 528 returns 0 for error.  Other similar new lines (538, 549, etc.) return ret.  Why is new line 528 different?

Reply:
Fixed. Nice catch!